### PR TITLE
style(code): fixing indent

### PR DESCRIPTION
**Pull request by iam4x**
Hey,

Maybe my smallest PR ever made, one whitepsace remove, juste to stick the code style rules ;)

### CODE REVIEW

File: lib/inline-view.coffee [hunk 1]
```diff
@@ -32,7 +32,7 @@ class InlineView
     # Config value if you want to limit the status bar report
     # if your cursor is in the range or error, or on the line
     limitOnErrorRange = (atom.config.get('linter.statusBar') ==
-                         'Show error if the cursor is in range')
+                        'Show error if the cursor is in range')
```
Comment (dmnd) [inline lib/inline-view.coffee]: I actually think the origin indent is correct! [See PEP8](https://www.python.org/dev/peps/pep-0008/#indentation) for a reference

Comment (iam4x) [inline lib/inline-view.coffee]: @dmnd `Coffeelint` grunt task gave me an error. I suggest an other change in my new commit to fit for everyone ;)

**Comment by iam4x** [comment]
@dmnd I presume you are okey with it now?

**Comment by dmnd** [comment]
:+1:

Sorry, I didn't realise coffeelint considered PEP8 style indenting an
error. How annoying :)

If this passes lint I'm cool with it.

**Comment by iam4x** [comment]
And it's done!

**Pull request merged**